generators/server/templates/dotnetcore/src/Project/Infrastructure/IdentityStartup.cs.ejs

```diff
@@ -25,17 +25,19 @@ namespace <%= namespace %>.Infrastructure {
         public static IApplicationBuilder UseApplicationIdentity(this IApplicationBuilder @this,
             IServiceProvider serviceProvider)
         {
-            var userManager = serviceProvider.GetRequiredService<UserManager<User>>();
-            var roleManager = serviceProvider.GetRequiredService<RoleManager<Role>>();
+            using (var scope = serviceProvider.CreateScope())
+            {
+                var userManager = scope.ServiceProvider.GetRequiredService<UserManager<User>>();
+                var roleManager = scope.ServiceProvider.GetRequiredService<RoleManager<Role>>();
 
-            //            var op1 = new Operation {Date = DateTime.Now, Description = "An operation", Amount = (decimal) 10.0};
-//            var op2 = new Operation {Date = DateTime.Now, Description = "Another operation", Amount = (decimal) 20.0};
-//            var op3 = new Operation {Date = DateTime.Now, Description = "Last one operation", Amount = (decimal) 30.0};
+                //            var op1 = new Operation {Date = DateTime.Now, Description = "An operation", Amount = (decimal) 10.0};
```

Comment:
Can be removed, it certainly represents early tests I did with  BankAccount/ Operation domain classes